kvbc/include/KVBCInterfaces.h

```diff
@@ -65,7 +65,8 @@ class IClient {
                                     std::chrono::milliseconds timeout,
                                     uint32_t replySize,
                                     char* outReply,
-                                    uint32_t* outActualReplySize) = 0;
+                                    uint32_t* outActualReplySize,
+                                    std::string cid = "") = 0;
```

Comment:
Maybe take an rvalue-ref and move the string? I'm not sure if this matters, since the cid is likely small. If it's going to be copied into a message anyway, though, it should be taking by string ref.

Reply:
FIxed